index.html

```diff
@@ -6,35 +6,101 @@
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap"
+      rel="stylesheet" />
   </head>
-  <body class="m-5 text-center">
-    <main class="container d-flex flex-column align-items-center">
-      <h1 class="m-4">Awesome Books</h1>
-      <table class="table table-striped border border-dark border-3">
-        <tbody id="book-list"></tbody>
-      </table>
-      <div class="row justify-content-center">
-        <hr class="col-1" />
-      </div>
-      <form id="form" class="d-flex flex-column w-50">
-        <h2>Add a new book</h2>
-        <div class="form-group">
-          <input type="text" class="form-control my-3" placeholder="Title" />
+  <body>
+    <header class="text-center mt-2">
+      <nav class="navbar navbar-expand-lg navbar-light px-4 border
+        border-2 border-dark">
+        <a class="navbar-brand" href="#">Awesome Books</a>
+        <button class="navbar-toggler" type="button"
+          data-toggle="collapse"
+          data-target="#navbarNavAltMarkup"
+          aria-controls="navbarNavAltMarkup"
+          aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav ms-auto">
+            <a id="list-nav-item" class="nav-item nav-link active border-2
+              border-end border-dark"
+              href="#">List</a>
+            <a id="add-nav-item" class="nav-item nav-link border-2
+              border-end border-dark" href="#">Add
+              new</a>
+            <a id="contact-nav-item" class="nav-item nav-link"
+              href="#">Contact</a>
+          </div>
         </div>
-        <div class="form-group">
-          <input type="text" class="form-control my-3" placeholder="Author" />
+      </nav>
+      <span id="date" class="float-end me-5 mt-2"></span>
+    </header>
+    <main class="container p-5">
+      <section id="list-section" class="text-center">
+        <h1 class="m-4">Awesome Books</h1>
+        <table class="table table-striped border border-dark border-3">
```

Comment:
- [ ] Let's make some adjustments to this section, please. It would be nice if it aligned with the wireframe we've been given.

| The wireframe |
| --- |
| ![table-wireframe](https://user-images.githubusercontent.com/45002654/133660683-a59abfe7-16b0-42c0-9c12-df1020698f77.PNG) |

| Your implementation |
| --- |
|  ![table-imp](https://user-images.githubusercontent.com/45002654/133660622-59cea59e-57cf-43ad-88bc-1318f050caf1.PNG) |

The text and the button are aligned along the right and left margins of the table. Please add some CSS styles to address this issue.